doc/covid-response.Rmd

```diff
@@ -0,0 +1,106 @@
+
+The team feels that the question presents an opportunity to meaningfully explore COVID-19 data through a simple statistical analysis and stays true to the project scope as discussed in the section above. The question provides a stepping stone towards helping develop a prefatory understanding of COVID-19 responses in Canada and USA and can be built upon further through more robust analyses. The team acknowledges that there are several assumptions and limitations associated with this analysis and they are discussed in the relevant sections below. 
+
+## Data
+
+The data set used in this project comes from the `Our World in Data COVID-19` database created by Hannah Richie et al. [@owidcoronavirus]. This data set examines the impact of COVID-19 on countries all over the world; the data set comprises daily statistics pertaining to the pandemic from over 200 countries recorded since December 31st 2019. Each row in the data set represents a date in a country, where measurements like total cases, new daily cases, hospital admission rates etc. are recorded. Data has been collected in conjunction with the World Health Organization (WHO), the European Center for Disease Prevention and Control (ECDC) and is available on [Our World in Data](https://ourworldindata.org/coronavirus) and raw data can be found [here](https://raw.githubusercontent.com/owid/covid-19-data/master/public/data/owid-covid-data.csv).
+
+## Assumptions and limitations
+
+1. We've restricted the time-line for consideration to be between March 1st 2020 and October 31st 2020.
+2. The choice of timeline is further explained in our Exploratory Data Analysis reports under the eda folder.
+3. We acknowledge that it is impossible to gather data regarding the true population of confirmed COVID-19 cases and the viral detection tests being conducted.
+4. Our sample sizes for both Canada and USA while calculating daily response ratios are limited by the data as present in our data set. 
+5. We acknowledge that the nature of our data set is temporal and that there will be autocorrelation within each country's response rate calculations.
+6. Our analysis seeks to establish a simple comparison with no attempts to infer causality and/or other overarching conclusions about either 
+
+## Methodology
+
+The team performed extensive EDA to wrangle the data and make it suitable for further analysis. Details of the EDA process are captured in our reports under the eda folder which can be found [here](https://github.com/UBC-MDS/covid-19-cases-vs-tests-analysis/tree/main/eda).
+
+Firstly, we calculated the daily response ratio for both Canada and USA for the time-line of interest using the daily new_tests and new_cases columns found in the data set. We analyzed the distribution of the daily response ratio for both the countries during the time-line of interest as shown in the figure below.
```

Comment:
Should it look better if we wrap new_tests and new_cases with backticks since we're referring to them as "columns" of the data set? 

> Firstly, we calculated the daily response ratio for both Canada and USA for the time-line of interest using the daily `new_tests` and `new_cases` columns ...

Also, Grammarly is showing that we're missing the "the" article before "USA" here inn this paragraph 😄

Reply:
Acknowledged and corrected throughout.